ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -497,7 +497,7 @@ function admin_ack() {
 
     echo "Require admin ack"
     local wait_time_loop_var=0 ack_data
-    ack_data="$(echo ${out} | awk '{print $2}' | cut -f2 -d\")" && echo "Admin ack patch data is: ${ack_data}"
+    ack_data="$(echo ${out} | awk -F'"' '{print $2}')" && echo "Admin ack patch data is: ${ack_data}"
     oc -n openshift-config patch configmap admin-acks --patch '{"data":{"'"${ack_data}"'": "true"}}' --type=merge
```

Comment:
Can we output the command for debug?

Reply:
You mean output for this command only or all the command in cucushift-upgrade-toimage? actually `echo "Admin ack patch data is: ${ack_data}"` is enough for debug if only for this command.

Reply:
look at https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.14-arm64-stable-4.14-upgrade-from-stable-4.13-azure-ipi-disconnected-fullyprivate-f28/1713794938363187200/artifacts/azure-ipi-disconnected-fullyprivate-f28/cucushift-upgrade-toimage/build-log.txt

Seems `Admin ack patch data is: 1.27` is not enough.

Reply:
I mean only this command, #501

Reply:
Then you mean we should output ${out}, right?

Reply:
no, I want to know what we ran when debugging the failures.

Reply:
> oc -n openshift-config patch configmap admin-acks --patch '{"data":{"'"${ack_data}"'": "true"}}' --type=merge

hmmm, only ${ack_data} is what we cared in above command. so i'm not quite getting u here, why ` echo "Admin ack patch data is: ${ack_data}"` is not enough for debug? could u explain more your requirement here:)

Reply:
All right, then that's enough for debugging.